odata-url-conventions/5 Query Options.md

```diff
@@ -2017,7 +2017,8 @@ properties of the identified instances of a structured type, optionally followed
 related entity or entities, optionally followed by a [type-cast segment](#AddressingDerivedTypes)
 to expand only related entities of that derived type or one of its
 sub-types, optionally followed by `/$ref` to expand only entity
-references.
+references, or `/$count`, optionally with expand options, to return only the count of 
```

Comment:
```suggestion
references, or `/$count`, optionally with expand options [#OData-Protocol#ExpandOptions], to return only the count of 
```

Reply:
Implemented as:
```suggestion
references, or `/$count`, optionally with [expand options](#OData-Protocol#ExpandOptions), to return only the count of 
```

is that what you had in mind (i.e., the `expand options` a hot link)?

Reply:
Yes.